eppo_core/src/eval/eval_precomputed_assignments.rs

```diff
@@ -0,0 +1,205 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use chrono::{DateTime, Utc};
+
+use crate::eval::get_assignment;
+use crate::ufc::{AssignmentFormat, Environment};
+use crate::{error::EvaluationFailure, ufc::Assignment};
+use crate::{Attributes, Configuration, EvaluationError, Str};
+
+#[derive(Debug)]
+pub struct PrecomputedConfiguration {
+    pub created_at: DateTime<Utc>,
+    pub format: AssignmentFormat,
+    pub environment: Environment,
+    pub flags: HashMap<String, Result<Assignment, EvaluationFailure>>,
+}
+
+impl PrecomputedConfiguration {
+    pub fn empty(environment_name: &str) -> Self {
+        Self {
+            created_at: Utc::now(),
+            format: AssignmentFormat::Precomputed,
+            environment: Environment {
+                name: environment_name.into(),
+            },
+            flags: HashMap::new(),
+        }
+    }
+}
+
+pub fn get_precomputed_assignments(
+    configuration: Option<&Configuration>,
+    subject_key: &Str,
+    subject_attributes: &Arc<Attributes>,
+    early_exit: bool,
```

Comment:
Its always false because this is `eppo-core` and the only user is the edge function but why not let the caller pass whatever they desire? Are you saying because there is not use-case right now for early exit then don't bother implementing it?

Reply:
Yes. There's no use case now and there's a good chance there'll never be